tracer/contrib/sqltraced/sqlutils/testing.go

```diff
@@ -7,6 +7,7 @@ import (
 	"testing"
```

Comment:
This file is also concerned (and tests get packaged with it). Make it `sqlutils_test.go` instead.